index.html

```diff
@@ -16,59 +17,97 @@
 
 <body>
 
-  
-     
-  <div class="container home">
+  <div class="top">
+    
+      <h1>Fun Bus</h1>
+      <nav>
+        <a href="#">Home</a>
+        <a href="#">About Us</a>
+        <a href="#">Blog</a>
+        <a href="#">Contact</a>
+      </nav>
    
+  </div>
+  
+  <div class="lines">
```

Comment:
This div isn't really necessary. You could just put a `bottom-border: 2px dashed color` on the <div> with the top class.